framework/branch-key-store-admin.md

```diff
@@ -0,0 +1,185 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Key Store Admin
+
+The Key Store Administrative Client,
+or just "Key Store Admin".
+
+## Version
+
+1.0.0
+
+### Changelog
+
+- 1.0.0
+  - [Initial record](../changes/2025-01-17_key-store-admin/background.md)
+
+## Implementations
+
+| Language | Confirmed Compatible with Spec Version | Minimum Version Confirmed | Implementation |
+| -------- | -------------------------------------- | ------------------------- | -------------- |
+| Dafny    | TBD                                    | TBD                       | TBD            |
```

Comment:
update this matrix

Reply:
A little hard right now, since it is not released anywhere.
Thus, the matrix is technically accurate...